workspaces/ui/src/utilities/useAnalyticsHook.js

```diff
@@ -72,6 +73,9 @@ export function AnalyticsContextStore({ children, specService }) {
       const clientContext = {
         clientAgent: clientAgent,
         clientId: clientId,
+        //platform: os.platform(),
```

Comment:
Should there be any changes to this file?

Reply:
Thanks, good catch. No. Reverted in upcoming push.